modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -70,6 +77,7 @@ public static void testCleanup() {
     @Test
     public void test() throws SQLException {
         performSimpleTest(jdbcUrl);
+        performContainerStatusCheck(jdbcUrl);
```

Comment:
Since we test just a single case of TC_DAEMON here, maybe it worth to create a separate test, without Parameterized runner?